tool/args.cc

```diff
@@ -71,16 +70,20 @@ bool ParseKeyValueArguments(std::map<std::string, std::string> *out_args,
   return true;
 }
 
-void PrintUsage(const struct argument *templates) {
+void PrintUsage(const argument_t *templates) {
   for (size_t i = 0; templates[i].name[0] != 0; i++) {
-    const struct argument *templ = &templates[i];
+    const argument_t *templ = &templates[i];
     fprintf(stderr, "%s\t%s\n", templ->name, templ->description);
   }
 }
 
 bool GetUnsigned(unsigned *out, const std::string &arg_name,
                  unsigned default_value,
+#if !defined(OPENSSL_BENCHMARK)
```

Comment:
Why is this needed when: `typedef std::map<std::string, std::string> args_map_t;`?